default/python/AI/character/character_module.py

```diff
@@ -379,6 +383,9 @@ def may_dither_focus_to_gain_research(self):
     def may_research_heavily(self):
         return self.aggression > fo.aggression.cautious
 
+    def may_use_growth_focus(self):
+        return self.aggression >= fo.aggression.typical
```

Comment:
So, probably only changing this occurrence of ```fo.aggression.typical``` to ```fo.aggression.beginner```, and perhaps adding a comment confirming the current intent is to allow for all but make it easy to customize.